SelfAndPeerEvaluations/utils.py

```diff
@@ -0,0 +1,439 @@
+import canvasapi
+from collections import defaultdict
+import csv
+from collections import deque
+import json
+import statistics
+import random
+
+
+class PartnerEvalQuizIndividualStats:
```

Comment:
- This class is big enough to be moved into its own file.
- Also need to add a doc string to the class